src/video.tsx

```diff
@@ -85,6 +85,7 @@ export class Video extends Component<VideoProps, VideoComponentState> {
   previousTime: number = -1;
   videoRef: RefObject<HTMLVideoElement> = React.createRef<HTMLVideoElement>();
   audioRef: RefObject<HTMLAudioElement> = React.createRef<HTMLAudioElement>();
+  onCanPlayTriggered: number = 0;
```

Comment:
I think this should be a boolean right? We are not using the number of calls for anything

Reply:
What? You didn't like my hipster 1 line check?

hehe sure, we can use binary...but my way was super slick! 👨‍🔬